content/hi/docs/reference/glossary/admission-controller.md

```diff
@@ -0,0 +1,21 @@
+---
+title: प्रवेश नियंत्रक
+id: admission-controller
+date: 2019-06-28
+full_link: /docs/reference/access-authn-authz/admission-controllers/
+short_description: >
+  कोड का एक टुकड़ा जो ऑब्जेक्ट की दृढ़ता से पहले कुबेरनेट्स एपीआई सर्वर के अनुरोधों को रोकता है।
+
+aka:
+tags:
+  - extension
+  - security
+---
+
+कोड का एक टुकड़ा जो ऑब्जेक्ट की दृढ़ता से पहले कुबेरनेट्स एपीआई सर्वर के अनुरोधों को रोकता है।
+
+<!--more-->
+
+प्रवेश नियंत्रक कुबेरनेट्स एपीआई सर्वर के लिए विन्यास योग्य हैं और "सत्यापन", "परिवर्तनशील", या दोनों हो सकता हैं। कोई भी प्रवेश नियंत्रक अनुरोध को अस्वीकार कर सकता है। उत्परिवर्तन नियंत्रक उन ऑब्जेक्ट्स को संशोधित कर सकते हैं जिन्हें वे स्वीकार करते हैं; सत्यापन नियंत्रक नहीं हो सकता है।
```

Comment:
So, there are a couple of things I have a problem with in this section.

The original text reads - Admission controllers are configurable for the Kubernetes API server and may be "validating", "mutating", or both. AFAIK, we are referring to the function of the controller in which case we need to change the grammar accordingly. If we're referring to the "quality" i.e. it is of type validating/mutating and how the admission controller can be both, we still need to change the grammar to reflect that. The current phrasing is a little ambiguous.

Also - mutating needs to have a standardized term instead of changing it in the first & third sentences within the section.

Reply:
@bishal7679 : This comment doesn't seem to have been addressed.